impls/factor/Dockerfile

```diff
@@ -24,8 +24,7 @@ WORKDIR /mal
 # Factor
```

Comment:
Please modify the top of the Dockerfile: change `FROM ubuntu:vivid` to `FROM ubuntu:20.04` (just to modernize the images).